lib/pgut/pgut-ipc.c

```diff
@@ -11,12 +11,10 @@
 
 #include <unistd.h>
 
-#ifdef HAVE_SYS_IPC_H
```

Comment:
Do we really need the change of the file?

Reply:
No, we don’t need change this file.